diff --exclude '*~' -ruN a/third_party/gpus/rocm_configure.bzl b/third_party/gpus/rocm_configure.bzl      
--- a/third_party/gpus/rocm_configure.bzl
+++ b/third_party/gpus/rocm_configure.bzl
@@ -181,22 +181,24 @@
       host compiler include directories, which can be added to the CROSSTOOL
       file.
     """
-    inc_dirs = []
+    #inc_dirs = []
+    #
+    ## Add HSA headers (needs to match $HSA_PATH)
+    #inc_dirs.append(rocm_config.rocm_toolkit_path + "/hsa/include")
+    #
+    ## Add HIP headers (needs to match $HIP_PATH)
+    #inc_dirs.append(rocm_config.rocm_toolkit_path + "/hip/include")
+    #
+    ## Add HIP-Clang headers (realpath relative to compiler binary)
+    #rocm_toolkit_path = realpath(repository_ctx, rocm_config.rocm_toolkit_path, bash_bin)
+    #inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/8.0/include")
+    #inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/9.0.0/include")
+    #inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/10.0.0/include")
+    #inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/11.0.0/include")
+    #
+    #return inc_dirs
 
-    # Add HSA headers (needs to match $HSA_PATH)
-    inc_dirs.append(rocm_config.rocm_toolkit_path + "/hsa/include")
-
-    # Add HIP headers (needs to match $HIP_PATH)
-    inc_dirs.append(rocm_config.rocm_toolkit_path + "/hip/include")
-
-    # Add HIP-Clang headers (realpath relative to compiler binary)
-    rocm_toolkit_path = realpath(repository_ctx, rocm_config.rocm_toolkit_path, bash_bin)
-    inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/8.0/include")
-    inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/9.0.0/include")
-    inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/10.0.0/include")
-    inc_dirs.append(rocm_toolkit_path + "/llvm/lib/clang/11.0.0/include")
-
-    return inc_dirs
+    return nixos sed target
 
 def _enable_rocm(repository_ctx):
     enable_rocm = get_host_environ(repository_ctx, "TF_NEED_ROCM")
